airbyte-integrations/connectors/destination-csv/src/test-integration/java/io/airbyte/integrations/destination/csv/CsvDestinationIntegrationTest.java

```diff
@@ -56,19 +54,27 @@ protected JsonNode getConfig() {
     return Jsons.jsonNode(ImmutableMap.of("destination_path", Path.of("/local").resolve(RELATIVE_PATH).toString()));
   }
 
-  @SuppressWarnings("ResultOfMethodCallIgnored")
+  // todo (cgardens) - it would be great if we could find a configuration here that failed. the
+  // commented out one fails in mac but not on the linux box that the github action runs in. instead
+  // we override the test here so it never runs.
   @Override
   protected JsonNode getFailCheckConfig() {
```

Comment:
one thing you can do is to remove write access to the directory

Reply:
`chmod -x directory`

Reply:
(this comment is for testCheckConnectionInvalidCredentials)

Reply:
is that not what `localRoot.toFile().setReadOnly();` does? (in the commented out code)?

Reply:
I don't know this API. The doc is a bit light so I don't know what it means in a unix environment, nor what it means on a directory. Is it changing permissions? Is it changing permissions for the user? the group? other? 🤷‍♂️ That's a very good example of abstraction that made things too simple to a point where you actually don't know what it does.

Reply:
kk. i'll circle back to this. i thought when i tried it out it removed all write permissions when i tried locally, but it was a little while ago so i don't remember.